share/shwap/pb/shwap_pb.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
```

Comment:
```suggestion

package share.p2p.shwap;
```
Or was package specifier ommited for some reason?

Reply:
It's not necessary from my understanding

Reply:
I don't think it's necessary, but other proto files specify the namespace so I expected this one too

Reply:
Ok. Also, note that the name of the pkg is more like an implementation detail. The formal spec won't contain the pkg name in its proto file.